web/vital_records/templates/vital_records/request/name.html

```diff
@@ -0,0 +1,18 @@
+{% extends "vital_records/base.html" %}
+{% load i18n %}
+{% block inner-content %}
+    <div class="row">
+        <div class="col-md-3">
+            <ol class="cagov-step-list">
+                <li data-status="{% if request.resolver_match.url_name == 'request_name' %}current{% endif %}">Name</li>
+                <li data-status="{% if request.resolver_match.url_name == 'request_detail' %}current{% endif %}">Preview</li>
+            </ol>
+        </div>
+        <div class="col-md-9">
+            <small>Step 1 of 6</small>
+            <h2>What is the name on the birth certificate?</h2>
+            <p class="mb-3">Please write the information as it appears on the birth certificate.</p>
+            {% include "vital_records/_two_column_form.html" with form=form button_label="Continue" %}
```

Comment:
This is really cool :+1:

Reply:
There are ways to make this even cooler! Stay tuned!